compiler/plugins/input/TOSA/InputConversion/test/apply_pdl_patterns_tosa.mlir

```diff
@@ -1,9 +1,9 @@
 // RUN: iree-opt --pass-pipeline="builtin.module(iree-preprocessing-apply-pdl-patterns{patterns-file=%p/tosa.pdl.mlir})" %s | FileCheck %s
 
-// CHECK-LABEL:   stream.executable private @mlp_external_executable
+// CHECK-LABEL:   stream.executable private @mlp_external_f32_f32_f32_i32_i32_i32_executable
 //       CHECK:   stream.executable.export public @mlp_external_entry_point
 //       CHECK:   builtin.module
-//       CHECK:     func.func private @mlp_external
+//       CHECK:     func.func private @mlp_external_f32_f32_f32_i32_i32_i32
```

Comment:
Sorry if I was confusing, but we need to change only the symbol name of the `stream.executable`. The `func.func` can still be `mlp_external`... Did you mean to change it as well?

Reply:
Yes, I changed it intentionally, since this is what the shared library links to, so it could cause name collision if for example we also had a i32 matmul that will take the same name? what do you think?

Reply:
Then the matcher should fail.... basically the `mlp_external` is the call we need to produce. Thats the contract of the plugin... so we need to make the compiler match that contract... the contract also imlpies it needs to match the data types properly.

Reply:
ah cool, will change it back.